index.js

```diff
@@ -18,7 +18,11 @@ function validateScope(required, provided) {
   let hasScope = false;
 
   required.map(scope => {
```

Comment:
would you mind making this map a forEach since it doesn't need the return value?

Reply:
actually i can do it!